qiskit/schemas/qobj_schema.json

```diff
@@ -866,6 +901,22 @@
                         "init_qubits": {
                             "description": "Whether to initialize the qubits in the ground state before each shot.",
                             "type": "boolean"
+                        },
+                        "calibrations": {
+                            "description": "User-specified pulse-level calibrations for operations utilized in this experiment.",
+                            "type": "object",
+                            "properties": {
+                                "gates": {
+                                    "$ref": "#/definitions/calibration_entry"
+                                },
+                                "pulse_library": {
+                                    "description": "Pulses for gate calibrations.",
+                                        "items": {
+                                            "$ref": "#/definitions/pulse_library"
+                                        },
+                                    "type": "array"
+                                }
+                            }
```

Comment:
I'm a little confused, why should `pulse_library` be repeated here? Doesn't it already exist elsewhere in the qobj?

Reply:
oh i see we can't mix pulseqobj things into qasmqobj?

Reply:
Yes I think so, this is config for qasm qobj